public/index.html

```diff
@@ -35,7 +35,10 @@
 
     <!-- Add Lottie support -->
     <script src="https://unpkg.com/@lottiefiles/lottie-player@latest/dist/lottie-player.js"></script>
-
+    
+    <!-- Add Cookiescript consent manager -->
+    <script type="text/javascript" charset="UTF-8" src="//cdn.cookie-script.com/s/352d41aa6940aa9f18ef013ba45a515d.js"></script>
```

Comment:
Doesn't look like it, but just to confirm @conradbielitz : this does not need a JS dependency to be installed? Seems to work fine on the test deployment, but had to ask.

Reply:
Correct, this fetches and executes a JavaScript file from cookie-script.com.
There's no need to install any JavaScript dependency via npm, yarn, or any other package manager for this script to work. The browser will automatically download and run the script when the HTML page loads.

Reply:
Lovely, thanks!